workos/resources/organizations.py

```diff
@@ -26,3 +27,39 @@ def to_dict(self):
         organization = super(WorkOSOrganization, self).to_dict()
 
         return organization
+
+
+class WorkOSOrganizationList(WorkOSBaseResource):
+    """Representation of a list of WorkOS Organizations as returned by WorkOS through the Organizations feature.
+
+    Attributes:
+        OBJECT_FIELDS (list): List of fields a WorkOSOrganizationList is comprised of.
+    """
+
+    OBJECT_FIELDS = [
+        "data",
+        "list_metadata",
+    ]
+
+    @classmethod
+    def construct_from_response(cls, response):
+        return super(WorkOSOrganizationList, cls).construct_from_response(response)
+
+    def to_dict(self):
+        organizations = super(WorkOSOrganizationList, self).to_dict()
+
+        return organizations
+
+    def auto_paging_iter(self):
```

Comment:
If we're going to add auto-pagination we should switch to using our explicit `order` parameter for all the paginating methods so that we can get the new and improved pagination behavior.

This will mean that we can always use `after` to paginate deeper down the list.

Reply:
Hey @maxdeviant I just refactored this PR to use the order parameter so that we can take advantage of the new behavior and always use the "after" marker to return the next page. 

In order to make sure auto-pagination operates as expected when there is no order parameter submitted, I did make a change to the Python SDK list methods so that it will default to using order=Order.Desc. This allows users to call a list method without defining an order and still be able to auto-paginate. 

If a user is manually paginating and not specifying an order, this could be a breaking change. I'm of the mind that we should push people into the new behavior anyway, so maybe this is a good opportunity to do that. 

Do you have any thoughts about updating the SDK's to use Order.Desc as the default order param if none is submitted to opt everyone in to the new behavior?